extend.cpp

```diff
@@ -1530,25 +1532,18 @@ namespace Sass {
         }
       }
 
-
-
-//      DEBUG_EXEC(EXTEND_COMPOUND, printCompoundSelector(pSels, "SELS: "))
-
+      DEBUG_EXEC(EXTEND_COMPOUND, printCompoundSelector(pSels, "SELS: "))
```

Comment:
I guess it doesn't really matter. Only has any effect if DEBUG is on, and I needed to debug that information at some point, so I started to uncomment them (there are already other such statements that are uncommented). IMO it can stay that way (more debug info enabled)